scripts/sonic-kdump-config

```diff
@@ -826,11 +826,15 @@ def main():
     parser.add_argument('--num_dumps', nargs='?', type=int, action='store', default=False,
         help='Maximum number of kernel dump files stored')
 
+    parser.add_argument('--remote', action='store_true', default=False,
+                help='ssh_string for remote kdump')
```

Comment:
Please fix the help string. This string is the same as the help string for the option "-ssh_string" added below.